packages/@sanity/form-builder/src/inputs/PortableText/_legacyDefaultParts/__tests__/BlockExtras.test.tsx

```diff
@@ -0,0 +1,139 @@
+        marks: [],
+      },
+    ],
+  },
+  {
+    _type: 'myTestBlockType',
+    _key: 'b',
+    style: 'normal',
+    markDefs: [],
+    children: [
+      {
+        _type: 'span',
+        _key: 'b1',
+        text: 'Yo',
+        marks: [],
+      },
+    ],
+  },
+]
+const noop = () => {
+  // noop
+}
+const subscribe = () => noop
+
+function renderCustomMarkers(markers) {
+  return markers.map((marker, index) => {
+    if (marker.type === 'customMarkerTest') {
+      return (
+        <div key={`marker-${index}`} data-testid="custom-marker-test">
+          Marker!
+        </div>
+      )
+    }
+    return null
+  })
+}
+
+function renderBlockActions() {
+  return <div data-testid="block-action-test">Action!</div>
+}
+
+const markers = [{type: 'customMarkerTest', path: [{_key: value[0]._key}]}]
+
+function renderInput(props: Partial<PortableTextInputProps> = {}) {
+  const onBlur = jest.fn()
+  const onFocus = jest.fn()
+  const onChange = jest.fn()
+
+  const {queryByTestId, queryAllByTestId, queryByText, findByText, findByTestId} = render(
+    <ThemeProvider scheme="light" theme={studioTheme}>
+      <LayerProvider>
+        <ToastProvider>
+          <TestInput
+            focusPath={[]}
+            type={schema.get('body')}
+            onBlur={onBlur}
+            onFocus={onFocus}
+            onChange={onChange}
+            markers={markers}
+            level={0}
+            renderCustomMarkers={renderCustomMarkers}
+            renderBlockActions={renderBlockActions}
+            readOnly={false}
+            presence={[]}
+            subscribe={subscribe}
+            value={value}
+            schema={schema}
+            {...(props || {})}
+          />
+        </ToastProvider>
+      </LayerProvider>
+    </ThemeProvider>
+  )
+  return {onChange, onFocus, queryByTestId, queryAllByTestId, queryByText, findByText, findByTestId}
+}
+
+describe('Portable Text Editor Block Extras', () => {
+  test('custom markers', async () => {
+    const {findByText, queryByTestId} = renderInput()
+    const block = await findByText('Lorem ipsum dolor sit amet', {exact: false})
+    if (block) {
+      fireEvent.mouseOver(block)
+    }
+    await waitFor(() => expect(queryByTestId('custom-marker-test')).toBeTruthy())
+  })
+  // test('block actions', async () => {
+  //   const {queryByTestId, findByText} = renderInput()
+  //   const block = await findByText('Lorem ipsum dolor sit amet', {exact: false})
+  //   if (block) {
+  //     fireEvent.click(block)
+  //   }
+  //   await waitFor(() => expect(queryByTestId('block-action-test')).toBeTruthy())
+  // })
```

Comment:
Would prefer `test.skip` here, instead of commenting out, so it’s easier to find later.